test-case/check-sof-logger.sh

```diff
@@ -75,7 +75,7 @@ size=`du -k $tmp_file | awk '{print $1}'`
 fw_log_err=`grep -i "error" $tmp_file`
 # 100 here is log header size
```

Comment:
please update the comment accordingly as well.

Reply:
done